services/gateway/app.js

```diff
@@ -109,7 +109,7 @@ tempApp.run().then(async () => {
 			use: [],
 
 			cors: {
-				origin: '*',
+				origin: config.cors.allowedOrigin,
```

Comment:
We should be able to support multiple origins.
Something like what is done here https://github.com/moleculerjs/moleculer-web/issues/43
```suggestion
				origin: config.cors.allowedOrigin.split(","),
```

Reply:
Applied.